docs/en/sql-reference/functions/tuple-map-functions.md

```diff
@@ -5,40 +5,72 @@ toc_title: Working with maps
 
 # Functions for maps {#functions-for-working-with-tuple-maps}
 
-## mapAdd(Tuple(Array, Array), Tuple(Array, Array) [, ...]) {#function-mapadd}
+## mapAdd {#function-mapadd}
 
 Collect all the keys and sum corresponding values.
 
-Arguments are tuples of two arrays, where items in the first array represent keys, and the second array contains values for the each key.
-All key arrays should have same type, and all value arrays should contain items which are promotable to the one type (Int64, UInt64 or Float64).
+**Syntax** 
+
+``` sql
+mapAdd(Tuple(Array, Array), Tuple(Array, Array) [, ...])
+```
+
+**Parameters** 
+
+Arguments are [tuples](../../sql-reference/data-types/tuple.md#tuplet1-t2) of two [arrays](../../sql-reference/data-types/array.md#data-type-array), where items in the first array represent keys, and the second array contains values for the each key.
+All key arrays should have same type, and all value arrays should contain items which are promotable to the one type ([Int64](../../sql-reference/data-types/int-uint.md#int-ranges), [UInt64](../../sql-reference/data-types/int-uint.md#uint-ranges) or [Float64](../../sql-reference/data-types/float.md#float32-float64)).
 The common promoted type is used as a type for the result array.
```

Comment:
If this sentece is supposed to be the separate paragraph, you must add the new line before it. Otherwise it is not perceived as a separate paragraph.

What is it common promoted type? Is it the resulting common type? Promotion is possibly the worst word that can be used in the meaning of type conversion.

Reply:
Here and further.

Reply:
Replaced with "reduced"